src/scales/scale.logarithmic.js

```diff
@@ -144,26 +145,26 @@ module.exports = function(Chart) {
 					var meta = chart.getDatasetMeta(datasetIndex);
 					if (chart.isDatasetVisible(datasetIndex) && IDMatches(meta)) {
 						helpers.each(dataset.data, function(rawValue, index) {
-							var value = +me.getRightValue(rawValue);
+							var value = me.parseValue(rawValue);
 							// invalid, hidden and negative values are ignored
-							if (isNaN(value) || meta.data[index].hidden || value < 0) {
+							if (isNaN(value.val) || meta.data[index].hidden || value.val < 0) {
```

Comment:
validate `min` and `max` instead of `val`

Reply:
done